sound/soc/sof/intel/hda-loader.c

```diff
@@ -35,7 +35,7 @@ static struct hdac_ext_stream *cl_stream_prepare(struct snd_sof_dev *sdev, unsig
 	struct pci_dev *pci = to_pci_dev(sdev->dev);
 	int ret;
 
-	dsp_stream = hda_dsp_stream_get(sdev, direction);
+	dsp_stream = hda_dsp_stream_get(sdev, direction, false);
```

Comment:
Would some predefined macro here to replace the `false` to improve the code readability.
```suggestion
	dsp_stream = hda_dsp_stream_get(sdev, direction, DMI_L1_NOT_COMPATIBLE);
```

Reply:
Hmm, I do like this. We have many calls to stream_get() and this version is much more readable. The definition side is a bit akward, but maybe something like:
```
#define DMI_L1_NOT_COMPATIBLE false
#define DM_L1_COMPATIBLE true
```

Not pretty, but helps readability of the functions for sure.

Reply:
I personally dont like this macro suggestion. Maybe the true/false dont tell you what they are for but it takes one glance at the function signature to deduce that. But, I will do it if everyone feels the need for this. @plbossart @lyakh thoughts?

Reply:
I like the define not really because of the readability but the extensibility. If later on you want to add a flag in some cases, it easier. But we've need a prefix to avoid conflicts.

Reply:
Thanks @plbossart. thats definitely a lot better. changed now